hca/upload_to_cloud.py

```diff
@@ -89,6 +89,8 @@ def upload_to_cloud(files, staging_bucket, replica, from_cloud=False):
                     "hca-dss-content-type": _mime_type(fh.raw.name)
                 }
 
+                raw_fh.close()
```

Comment:
Not sure I follow. It's an open binary file that needs to be closed when it's done being used. What discrepancy are you seeing?